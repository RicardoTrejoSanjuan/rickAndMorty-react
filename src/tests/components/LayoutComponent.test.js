import React from 'react';
const { shallow } = require("enzyme");
const { default: Layout } = require("../../components/LayoutComponent");


describe('Test en <Layout />', () => {
    
    test('Dede mostrar el componente <Layout/>', () => {

        const wrapper = shallow(<Layout/>);

        expect(wrapper).toMatchSnapshot();
    });

    test('Debe contener los siguientes elementos html', () => {

        const wrapper = shallow(<Layout />);

        expect(wrapper.find('nav').exists()).toBe(true);
        expect(wrapper.find('.container').exists()).toBe(true);
        expect(wrapper.find('button').exists()).toBe(true);
        expect(wrapper.find('.nav-item').exists()).toBe(true);
    });

});