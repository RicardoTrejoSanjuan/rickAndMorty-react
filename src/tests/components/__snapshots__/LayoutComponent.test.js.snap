// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <Layout /> Dede mostrar el componente <Layout/> 1`] = `
<Fragment>
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <div
      className="container"
    >
      <Link
        className="navbar-brand"
        to="/rickAndMorty-react/"
      >
        Home
      </Link>
      <button
        aria-controls="navbarNav"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-target="#navbarNav"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse"
        id="navbarNav"
      >
        <ul
          className="navbar-nav"
        >
          <li
            className="nav-item"
          >
            <Link
              className="nav-link"
              to="/rickAndMorty-react/about"
            >
              About
            </Link>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</Fragment>
`;
